src/effects.js

```diff
@@ -0,0 +1,18 @@
+var old = jQuery.Tween.prototype.run;
+
+jQuery.Tween.prototype.run = function( percent ) {
+	if ( jQuery.easing[ this.easing ].length > 1 ) {
+		migrateWarn(
```

Comment:
We need an entry in the `warnings.md` file explaining the message and what to do about it.